test-bench-utils/lib/sinks/crypto.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+const crypto = require('crypto');
+
+/**
+ * @param {string} not used but need to keep signature for abstracted helpers
+ * @param {Object} opts
+ * @param {boolean=} opts.safe are we calling the sink safely?
+ * @param {boolean=} opts.noop are we calling the sink as a noop?
+ */
+module.exports['crypto-bad-mac'] = async function exec(
+  input,
+  { safe = false, noop = false } = {}
+) {
+  if (noop) return 'NOOP';
+
+  const algorithm = safe ? 'sha512' : 'md5';
+
+  return new Promise((resolve) => {
+    const hash = crypto.createHash(algorithm);
+    hash.update('salt');
+    const value = hash.digest('hex');
+    resolve(value);
+  });
+};
```

Comment:
This is in an `async function` so it's already being coerced to a `Promise`

```suggestion
  return crypto.createHash(algorithm)
    .update('salt')
    .digest('hex');
};
```